ui-tests/__test__/flows/feedback-form/render-vote-buttons.ts

```diff
@@ -0,0 +1,30 @@
+import { Page } from 'playwright/test';
+import { waitForAnimationEnd } from '../../helpers';
+import { openNewTab } from '../open-new-tab';
+import { closeTab } from '../close-tab';
+
+export const renderVoteButtons = async (page: Page, skipScreenshots?: boolean): Promise<void> => {
+  await closeTab(page, false, true);
+  await openNewTab(page, false, true);
+
+  await page.evaluate((body) => {
+    const selectedTabId = window.mynahUI.getSelectedTabId();
+    if (selectedTabId != null) {
+      window.mynahUI.updateStore(selectedTabId, {
+        chatItems: [],
+      });
```

Comment:
nit: this check looks redundant as we open a new tab, but ok with keeping it. In general I wonder whether it's better for the other new tests, to close tab and open tab at the beginning of the test, compared to resetting the chatItems in the store, not sure if there is a difference, I dont have strong opinions

Reply:
We've noticed that closing and opening tabs for each flow causes issues with flakiness and is unstable, so I try to only do it on the first one of a test group, and then only doing it if a full reset is actually necessary and can't be done easily through the store.

Reply:
But you're definitely right on that initial store update not being necessary, removed it.